Delta/Emulation/GameViewController.swift

```diff
@@ -1070,6 +1087,71 @@ extension GameViewController
             emulatorCore.rate = emulatorCore.deltaCore.supportedRates.lowerBound
         }
     }
+    
+    func performStatusBarAction()
+    {
+        let enabled = ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled
+        ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled = !enabled
+        
+        if let pauseView = self.pauseViewController
+        {
+            pauseView.dismiss()
+        }
+        
+        self.setNeedsStatusBarAppearanceUpdate()
+    }
+    
+    func performScreenshotAction()
+    {
+        guard let snapshot = self.emulatorCore?.videoManager.snapshot() else { return }
+        
+        let scale = ExperimentalFeatures.shared.gameScreenshots.scale != nil ? ExperimentalFeatures.shared.gameScreenshots.scale?.rawValue : 1.0
+        let size = CGSize(width: snapshot.size.width * scale!, height: snapshot.size.height * scale!)
+        
+        UIGraphicsBeginImageContextWithOptions(size, false, 1.0)
+        UIGraphicsGetCurrentContext()!.interpolationQuality = .none
+        
+        snapshot.draw(in: CGRect(origin: CGPoint.zero, size: size))
+        let scaledSnapshot = UIGraphicsGetImageFromCurrentImageContext()
+        
+        UIGraphicsEndImageContext()
+        
+        if ExperimentalFeatures.shared.gameScreenshots.saveToPhotos
+        {
+            UIImageWriteToSavedPhotosAlbum(scaledSnapshot!, nil, nil, nil)
+        }
+        
+        if ExperimentalFeatures.shared.gameScreenshots.saveToFiles
+        {
+            let data = scaledSnapshot!.pngData()
+            
+            let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+            let folderName = paths[0].appendingPathComponent("Screenshots")
+            try? FileManager.default.createDirectory(at: folderName, withIntermediateDirectories: true, attributes: nil)
+            
+            let date = Date()
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateFormat = "yyyy-MM-dd_HH-mm-ss"
+            
+            let fileName: URL
+            if let game = self.game
+            {
+                let gameDeepTitle = (game.fileURL.lastPathComponent as NSString).deletingPathExtension
+                fileName = folderName.appendingPathComponent(gameDeepTitle + "_" + dateFormatter.string(from: date) + ".png")
+            }
```

Comment:
If you want to get the game's ID (aka deep link title), I recommend this instead:

```suggestion
            if let game = self.game as? Game
            {
                let filename = game.identifier + "_" + dateFormatter.string(from: date) + ".png"
                fileName = folderName.appendingPathComponent(filename)
            }
```

However, since this is meant to be user-facing, I think it'd be better to use the Game's name instead, not its identifier. Will make it easier for users to find correct screenshot. So something like this:

```suggestion
            if let game = self.game as? Game
            {
                let filename = game.name + "_" + dateFormatter.string(from: date) + ".png"
                fileName = folderName.appendingPathComponent(filename)
            }
```